tests/cmdline/commands/test_calcjob.py

```diff
@@ -356,3 +356,42 @@ def test_calcjob_remotecat(self):
         options = [str(self.result_job.uuid), 'fileA.txt']
         result = self.cli_runner.invoke(command.calcjob_remotecat, options)
         assert result.stdout == 'test stringA'
+
+    def test_calcjob_gotocomputer(self, monkeypatch):
+        """Test verdi calcjob gotocomputer"""
+
+        from unittest.mock import patch
+
+        from aiida.common.exceptions import NotExistent
+
+        options = [str(self.result_job.uuid)]
+
+        # Easy peasy no exception
+        with patch('os.system') as mock_os_system:
+            result = self.cli_runner.invoke(command.calcjob_gotocomputer, options)
+            mock_os_system.assert_called_once()
+            assert mock_os_system.call_args[0][0] is not None
+
+        def raise_(e):
+            raise e
+
+        # Test when get_transport raises NotExistent
+        with patch(
+            'aiida.orm.nodes.process.calculation.calcjob.CalcJobNode.get_transport', new=lambda _: raise_(NotExistent)
+        ):
+            result = self.cli_runner.invoke(command.calcjob_gotocomputer, options)
+            assert result.exit_code == 1
```

Comment:
Would be good to check for an explicit error message, like I suggested below. Just don' know what this error would print

Reply:
Normally would print "No computer has been set for this calculation", 
but here prints nothing because the patch function prints nothing.
source code is:
```python
    try:
        transport = calcjob.get_transport()
    except NotExistent as exception:
        echo.echo_critical(repr(exception))
```

Reply:
ok, added "something" as the mocked message, just to check